pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -405,7 +405,7 @@ func (m *kubeGenericRuntimeManager) makeMounts(opts *kubecontainer.RunContainerO
 		} else {
 			fs.Close()
 
-			// Chmod is needed because ioutil.WriteFile() ends up calling
+			// Chmod is needed because os.WriteFile() ends up calling
```

Comment:
Here is using `m.osInterface.Create` which is using `os.Create()`, not `os.WriteFile()`.